components/unzipped/dashboard/DashboardTable.js

```diff
@@ -44,34 +36,48 @@ const Box = styled.div`
     align-items: center;
 `;
 
-const UserContainer = styled.div`
-    width: 100%;
-    display: flex;
-    flex-flow: column;
-`;
-
-const Row = styled.div`
-    display: flex;
-    flex-flow: row;
-    align-items: center;
-    position: relative;
-`;
-
 const StoryTable = styled.div``;
 
-const Panel = ({list, selectedList, type, projects=[], businesses, loading}) => {
-    const [menuOpen, setMenuOpen] = useState(false)
+const Panel = ({ type, businesses, loading, userType }) => {
 
-    const setDropdowns = (item) => {
-        setTimeout(function() { 
-            setMenuOpen(item)
-        }, 500);
-    }
+    const router = useRouter();
 
-    const setCloseDropdowns = (time) => {
-        setTimeout(function() { 
-            setMenuOpen(false)
-        }, (time || 500));
+    const generatePopout = (userType, item) => {
+        if (userType === 0 || userType === 2) {
+            return [
+                {
+                    text: 'Invoice',
+                    onClick: () => router.push(`projects/client/invoice/${item._id}`),
+                },
+                {
+                    text: 'Details',
```

Comment:
Can you please elaborate about these logs? if its not necessary then remove it

Reply:
These are already there.